### PR TITLE
CorrDiff API fix

### PR DESCRIPTION
**Pull request by NickGeneva**
# Earth2Studio Pull Request

## Description

Fixing an API break in newer versions of Modulus.

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/earth2studio/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/earth2studio/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/earth2studio/issues) is linked to this pull request.

### CODE REVIEW

File: earth2studio/models/dx/corrdiff.py [hunk 1]
```diff
@@ -329,12 +332,21 @@ def _forward(self, x: torch.Tensor) -> torch.Tensor:
...
         rnd = StackedRandomGenerator(x.device, sample_seeds)
+
+        # API break: https://github.com/NVIDIA/modulus/pull/540
+        if Version(modulus.__version__) <= Version("0.6.0"):
+            img_resolution_x = self.regression_model.img_resolution
+            img_resolution_y = self.regression_model.img_resolution
+        else:
+            img_resolution_x = self.regression_model.model.img_shape_x
+            img_resolution_y = self.regression_model.model.img_shape_y
+
```
Comment (dallasfoster) [inline earth2studio/models/dx/corrdiff.py]: I am sort of the opinion that this model should be frozen in time and note expected to be reusable with other domains or checkpoints. In that sense, I think we should simply fix `img_resolution_x` and `img_resolution_y` to be 448 and 448.

Comment (NickGeneva) [reply earth2studio/models/dx/corrdiff.py]: Good call, I updated it so it just uses the coords to spec this. So now the coords control the latent generation.

**Review by dallasfoster — APPROVED** [review]

**Pull request merged**